### DISCUSSION TITLE
Flash bootloader only

### DISCUSSION THREAD

**Discussion by mbilloo** [OE4T/meta-tegra#728] [Q&A] [answered]
Hi,

We're working on making changes to u-boot and it's a waste of time to flash all of the partitions for testing. I tried to modify the flash.xml.in to so that everything but the RFS is flashed (since that takes the longest), but that failed (during the flashing process, it looks like the entire Flash is erased first). Is there a way to only flash the bootloader? This is on a Jetson Nano eMMC.

Thanks

**Comment by madisongh** [comment, marked as answer]
To do this via USB, you should be able to use the `-c` option on the flash helper to write just the LNX partition (which is where U-Boot resides on the Nano) -  `./doflash.sh -c "sign; write LNX signed/boot.img.encrypt"`.

Another way would be to use the OTA bootloader update process - generate a BUP payload, copy it to the device,  and use the `l4t_payload_updater_t210` command on the Nano to install it.
